pkg/virt-operator/resource/generate/components/daemonsets.go

```diff
@@ -265,19 +266,18 @@ func NewHandlerDaemonSet(namespace, repository, imagePrefix, version, launcherVe
 	attachProfileVolume(pod)
 
 	bidi := corev1.MountPropagationBidirectional
-	// NOTE: the 'kubelet-pods-shortened' volume mounts the same host path as 'kubelet-pods'
-	// This is because that path holds unix domain sockets. Domain sockets fail when they're over
-	// ~ 100 characters, so that shortened volume path is to allow domain socket connections.
+	// NOTE: the 'kubelet-pods' volume mount exists because that path holds unix socket files.
+	// Socket files fail when their path is longer than 108 characters,
+	//   so that shortened volume path is to allow domain socket connections.
```

Comment:
tiny nit: redundant spaces

Reply:
It's a commenting style I like, those extra spaces show we're in the middle of a sentence.
I can delete them if that's against our coding style.

Reply:
Sure, thanks for elaborating :+1: